### ISSUE TITLE
Drop the cross-referencing of cached features; plot those with logits only, if need be.

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#85]
It looks like the most preferred neurons are generally off-distribution.

**Comment by DavidUdell** [comment]
Do this generally, I think: wherever there's an intersection op in place, substitute an operation that includes the relevant data when available, but otherwise simply continues without it.
